e2e/application/application_test.go

```diff
@@ -129,6 +129,7 @@ var ApplicationExecContext = func(context string, appName string) bool {
 var ApplicationPortForwardContext = func(context string, appName string) bool {
 	return ginkgo.It(context+": should get output of port-forward successfully", func() {
 		cli := fmt.Sprintf("vela port-forward %s 8080:80 ", appName)
+		time.Sleep(10 * time.Second) // Prevent port-forward to run when Current status=Pending, desired status=Running
```

Comment:
Might be better to use gomega's Eventually construct instead of hard coded sleep: https://onsi.github.io/gomega/#eventually